website/docs/d/public_ip.html.markdown

```diff
@@ -15,134 +15,16 @@ Use this data source to access the properties of an existing Azure Public IP Add
 
 ```hcl
 data "azurerm_public_ip" "datasourceip" {
-    name = "testPublicIp"
-    resource_group_name = "acctestRG"
+    name = "name_of_public_ip"
+    resource_group_name = "name_of_resource_group"
 }
 
-resource "azurerm_virtual_network" "helloterraformnetwork" {
-    name = "acctvn"
-    address_space = ["10.0.0.0/16"]
-    location = "West US 2"
-    resource_group_name = "acctestRG"
+output "public_ip_address_id" {
+    domain_name_label = "${data.azurerm_public_ip.datasourceip.id}"
```

Comment:
I may be wrong, but I think you're trying to output the domain name label rather than the ID (which isn't directly useful in an output IMO) - as such I think this needs to be:

```
output "domain_name_label" {
  value = "${data.azurerm_public_ip.datasourceip.domain_name_label}"
}
```